File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int cnt = 0;
        int n = in.nextInt();
        int[] array = new int[n];
        for(int i = 0; i < n; i++){
            array[i] = in.nextInt();
        }
        int f = in.nextInt();
        for(int i = 0; i < n; i++){
            if(array[i] == f)
                cnt++;
        }
        System.out.printf("%d",cnt);
    }
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var num = sc.nextInt()
    var arr = IntArray(num, {0})
    var cnt = 0
    for(i in 0..(num-1))
    arr[i] = sc.nextInt()
    var find = sc.nextInt()
    for(i in 0..(num-1))
    if(arr[i] == find)
    cnt++
    print(cnt)
}